src/main/java/Bro.java

```diff
@@ -0,0 +1,110 @@
+                if (isValidated1) {
+                    reply = new StringBuilder(" Marked " + tasks.get(indexOfMarkedTask) + " as done.");
+                    tasks.get(indexOfMarkedTask).setCompleted();
+                }
+                else {
+                    reply = new StringBuilder(TASK_DOES_NOT_EXIST);
+                }
+                break;
+            case "unmark":
+                int indexOfUnmarkedTask = Integer.parseInt(words[1]) - 1;
+                boolean isValidated2 = validateTask(tasks.size(), indexOfUnmarkedTask);
+                if (isValidated2) {
+                    reply = new StringBuilder(" Marked " + tasks.get(indexOfUnmarkedTask) + " as not done.");
+                    tasks.get(indexOfUnmarkedTask).setUncompleted();
+                }
+                else {
+                    reply = new StringBuilder(TASK_DOES_NOT_EXIST);
+                }
+                break;
+            case "todo":
+                Task todo = new ToDo(line);
+                tasks.add(todo);
+                reply = new StringBuilder(" added: " + todo);
+                break;
+            case "deadline":
+                StringBuilder deadlineName = new StringBuilder();
+                StringBuilder by = new StringBuilder();
+                int indexOfDeadline = Arrays.asList(words).indexOf("/by");
+                for (int i = 1; i < indexOfDeadline; ++i) {
+                    deadlineName.append(" ").append(words[i]);
+                }
+                for (int i = indexOfDeadline + 1; i < words.length; ++i) {
+                    by.append(" ").append(words[i]);
+                }
+                Task deadline = new Deadline(deadlineName.toString().trim(), by.toString().trim());
+                tasks.add(deadline);
+                reply = new StringBuilder(" added: " + deadline);
+                break;
+            case "event":
+                StringBuilder eventName = new StringBuilder();
+                StringBuilder startTime = new StringBuilder();
+                StringBuilder endTime = new StringBuilder();
+                int indexOfStartTime = Arrays.asList(words).indexOf("/from");
+                int indexOfEndTime = Arrays.asList(words).indexOf("/to");
+                for (int i = 1; i < indexOfStartTime; ++i) {
+                    eventName.append(" ").append(words[i]);
+                }
+                for (int i = indexOfStartTime + 1; i < indexOfEndTime; ++i) {
+                    startTime.append(" ").append(words[i]);
+                }
+                for (int i = indexOfEndTime + 1; i < words.length; ++i) {
+                    endTime.append(" ").append(words[i]);
+                }
+                Task event = new Event(eventName.toString().trim(), startTime.toString().trim(), endTime.toString().trim());
+                tasks.add(event);
+                reply = new StringBuilder(" added: " + event);
+                break;
+            }
```

Comment:
Can add default to avoid improper input.

Reply:
Thank you! I've added a `default` block in case the user inputs an invalid command.